docs/accepting-donations.rst

```diff
@@ -0,0 +1,75 @@
+========================
+Storage Server Donations
+========================
+
+The following is a configuration convention which allows users to anonymously support the operators of storage servers.
+Donations are made using `Zcash shielded transactions`_ to limit the amount of personal information incidentally conveyed.
+
+Sending Donations
+=================
+
+To support a storage server following this convention, you need several things:
+
+* a Zcash wallet capable of sending shielded transactions
+  (this *may* require a Zcash full node)
```

Comment:
Perhaps "(this currently requires a Zcash full node but may not in the future)"?